_includes/footer.html

```diff
@@ -1,6 +1,6 @@
 <footer>
     <h5>We love our sponsors!</h5>
-    <p>Please consider <a href="http://1418.team/sponsorship">sponsoring us</a>!</p>
+    <p style="color:white;">Please consider <a href="http://1418.team/sponsorship">sponsoring us</a>!</p>
```

Comment:
Avoid inline styling. Color this with CSS instead.

Reply:
How could I color just that line with CSS? When I use an internal style it changes all the paragraphs

Reply:
Add a class to the element (for example `.emphasized`) and then style that class.

Reply:
https://github.com/frc1418/frc1418.github.io/blob/master/css/main.sass#L377-L388

This is the relevant area you'll need to modify.